### ISSUE TITLE
Narrator doesn't read non-alphanumeric characters when focusing TextBox containing ones

### ISSUE THREAD

**Issue by stefansjfw** [microsoft/microsoft-ui-xaml#5477]
**Describe the bug**

When Narrator is on and user focuses on TextBox which Text contains non-alphanumerical characters, these characters are not read by Narrator. E.g. if TextBox.Text is "Shortcut is Win + `" Narrator would read it as "Shortcut is win plus".

On the other hand, while tipping non-alphanumerical in TextBox, Narrator reads them correctly.

**Steps to reproduce the bug**

1. Create any form including TextBox and any other element so focus could be switched between these two
2. Enter some text containing non-alphanumerical characters in the TextBox
3. Open Narrator
4. Switch focus between TextBox and other elements
5. Observe the issue while Narrator reads TextBox.Text when TextBox is focused

**Expected behavior**

Narrator reads non-alphanumerical characters in TextBox.

**Screenshots**

**Version Info**

NuGet package version: 
Microsoft.UI.Xaml 2.6.1

Windows app type:
| UWP              | Win32            |
| :--------------- | :--------------- |
|  |  |

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| October 2020 Update (19042)        |    |
| May 2020 Update (19041)            | Yes    |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            | Yes |
| Xbox               |  |
| Surface Hub        |  |
| IoT                |  |

**Additional context**

**Comment by StephenLPeters** [comment]
@chrisglein FYI

**Comment by chrisglein** [comment]
The whole value is there:
| property | value |
| --- | --- |
| Value.IsReadOnly: | false |
| Value.Value: | "Shortcut is Win + ``'" |

(can get this via inspect.exe)

The problem is with how Narrator reads that value. Which is outside the scope of this repo. Recommend filing feedback on Narrator via Feedback Hub.

**Comment by stefansjfw** [comment]
Submitted feedback via Feedback Hub, as suggested. Thanks
